client/termsetting/handlers/geneVariant.ts

```diff
@@ -35,12 +35,16 @@ export function getHandler(self: GeneVariantTermSettingInstance) {
 
 const idPrefix = `_geneVariant_AUTOID_${+new Date()}_`
 let id = 0
+const namePrefix = `_geneVariant_AUTONAME_${+new Date()}_`
+let name = 0
 
 export function fillTW(tw: GeneVariantTW, vocabApi?: VocabApi) {
 	if (!('id' in tw)) tw.id = idPrefix + id++
-	if (!tw.term.name && tw.term.isoform) tw.term.name = tw.term.isoform as string
+	if (!tw.term.name) tw.term.name = namePrefix + name++
```

Comment:
this will cause gene AKT1 to appear as `AKT11`

Reply:
The `tw.term.name` will equal `namePrefix + name++`. So the resulting name will be `_geneVariant_AUTONAME_<Date>_0`

Reply:
term.name is for displaying in term pill, so it cannot contain auto-incrementing value. we can introduce `term.id` for geneVariant and the id value will be auto-incrementing for uniquely identifying a geneV term